src/js/components/TaskListItemComponent.jsx

```diff
@@ -193,6 +195,16 @@ var TaskListItemComponent = React.createClass({
             {task.status}
           </span>
         </td>
+        <td className="text-center">
+          <TaskFileDownloadComponent task={task} fileName="stderr">
+            <i className="icon icon-mini file" /> stderr
+          </TaskFileDownloadComponent>
+        </td>
+        <td className="text-center">
+          <TaskFileDownloadComponent task={task} fileName="stdout">
+            <i className="icon icon-mini file" /> stdout
```

Comment:
Given that the file download text reads exactly like the `fileName` prop, and that the icon represents a file, why is this component responsible for the download icon and text nodes? Couldn't we simplify and move this into the TaskFileDownloadComponent itself? What do you think @orlandohohmeier ?